src/gui/properties/speedplotview.h

```diff
@@ -86,6 +90,36 @@ class SpeedPlotView : public QGraphicsView
     void paintEvent(QPaintEvent *event) override;
 
 private:
+    class Averager
```

Comment:
Better do the class implementation in the `speedplotview.cpp` file and leave the class definition here.

Reply:
Done